### PR TITLE
Implement functions for SRelativistic quantities.

### PR DESCRIPTION
**Pull request by dtabell**
This PR provides implementations of functions to compute accurate values
for a range of relativistic parameters and quantities — $\gamma$, $\gamma - 1$, $\beta$, $pc$, $E_\text{kin}$, $E_\text{tot}$,
and $B\rho$ — given values for the rest energy, $mc^2$, and $\beta\gamma$.

Items to address / comment on:
- [ ] function names
- [ ] function arguments — use E_\text{kin} instead of $\beta\gamma$?
- [ ] docstrings

The definitions given here will require corresponding changes to the tracking
routines. Should those changes be made part of this PR?

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -1,3 +1,125 @@
-function pc_ref(species::Species, beta_0)
-  return "hi!"
-end
\ No newline at end of file
+"""
+utilities.jl
+"""
```
Comment (mattsignorelli) [inline src/utils.jl]:
```suggestion
#=

Utility functions for tracking.

=#
```

**Comment by mattsignorelli** [comment]
Why is it simpler to use beta*gamma instead of just beta or gamma?

**Comment by dtabell** [comment]
> Why is it simpler to use beta*gamma instead of just beta or gamma?

@mattsignorelli Numerical stability: Given $\beta$, the computation of $\gamma$ for ultra-relativistic
particles loses accuracy. And given $\gamma$, the computation of $\beta$ for very-low-energy particles
loses accuracy. You can't avoid subtraction. On the other hand given $\beta\gamma$, or $E_\text{kin} = mc^2\beta\gamma$,
we have the very useful identity $\gamma^2 = 1 + (\beta\gamma)^2$, with which one can — regardless of energy
— compute accurate values for the other relativistic parameters and quantities .

**Comment by mattsignorelli** [comment]
Sounds good! Because of the discussion with @DavidSagan  #15 , I would say we can merge this for now and _not_ export the symbols, just so we can get tracking going, and then later decide where to let them live.

File: src/utils.jl [hunk 2]
```diff
@@ -1,3 +1,125 @@
...
+
+
+
+export sr_gamma,
+       sr_gamma_m1,
+       sr_beta,
+       sr_pc,
+       sr_ekin,
+       sr_etot,
+       brho
```
Comment (mattsignorelli) [inline src/utils.jl]:
```suggestion
```

**Comment by dtabell** [comment]
> @dtabell I have functions for computing these in `AcceleratorLattice`. We
should combine and put in `SimUtils` or perhaps `AtomicAndPhysicalConstants`.

@DavidSagan I'm puzzled as to why you need them in AcceleratorLattice. I did take
a look at your functions, and they are admirably flexible. But I rather like the idea of
forcing the user to give me $E_\text{kin}$ or $\beta\gamma$, because I know those allow me to compute
accurate value for everything else.

As to location, I think those functions belong in `BeamTracking` or, perhaps, `SimUtils`.
If `AcceleratorLattice` really needs them, then I favor `SimUtils`.

**Comment by dtabell** [comment]
@mattsignorelli I implemented both of your suggested changes.
(Did it by hand because I did not notice the `Commit suggestion` buttons above.)
I'm leaving in the file the commented-out functions that take $E_\text{kin}$ instead of $\beta\gamma$,
just in case we later choose to switch. We can delete them once we're certain.

**Review by mattsignorelli — APPROVED** [review]

**Pull request merged**